src/SFA.DAS.Forecasting.Jobs.Infrastructure/Configuration/EnvironmentVariables.cs

```diff
@@ -5,7 +5,7 @@ namespace SFA.DAS.Forecasting.Jobs.Infrastructure.Configuration
     public class EnvironmentVariables
     {
         //TODO: Get Connection String
```

Comment:
The todo was already there, but it looks out of date now, so we could remove it.